tests/components/quick_switch_modal.test.jsx

```diff
@@ -4,27 +4,27 @@
 import React from 'react';
 import {shallow} from 'enzyme';
 
+import {browserHistory} from 'utils/browser_history';
 import Constants from 'utils/constants.jsx';
 
 import QuickSwitchModal from 'components/quick_switch_modal/quick_switch_modal.jsx';
 
 describe('components/QuickSwitchModal', () => {
     const baseProps = {
+        getChannelUrlById: jest.fn(),
```

Comment:
You probably want to mock this in the actual test case so that it doesn't get called in one test and then have the other test case check if it was called.